clinica/cmdline.py

```diff
@@ -84,7 +83,6 @@ def cli(verbose: bool) -> None:
 cli.add_command(convert_cli)
 cli.add_command(generate_cli)
 cli.add_command(iotools_cli)
-cli.add_command(run_cli)
```

Comment:
That could probably stay as is, no? I used it as an example, but I think the dynamic registration pattern should start at the `clinica/pipelines/cli.py` level. WDYT?